spec/qiita/client_spec.rb

```diff
@@ -341,6 +341,10 @@ def stub_api_request
       204
     end
 
+    let(:response_hash) do
+      nil
+    end
```

Comment:
Returns empty body response if 204 status code.

https://github.com/bblimke/webmock/pull/190

And, Qiita `PUT` and `DELETE` APIs return a empty body.

JSON Schema(en): https://qiita.com/api/v2/schema?locale=en